apps/customquotes/manifest.yaml

```diff
@@ -0,0 +1,8 @@
+---
+id: custom-quotes
+name: Custom Quotes
+summary: Display custom quotes
+desc: Display quotes from a GSheet like this https://t.ly/rI5ZT.
```

Comment:
This link has "expired" and doesn't work. If possible please use a direct link rather than a shortener since those can stop working.

Reply:
Good catch. Thanks, updated it with direct link.